pkg/console/service/service_test.go

```diff
@@ -281,6 +288,7 @@ var _ = Describe("Service", func() {
 		Expect(json.NewDecoder(recorder.Body).Decode(tokenResponse)).To(Succeed())
 
 		Expect(tokenResponse.Token).To(Equal(testToken))
+		Expect(&tokenResponse.ExpirationTimestamp).To(Satisfy(testExpirationTimestamp.Equal))
```

Comment:
`Equal` matcher is not working here?

Reply:
No. `Equal` compares structs using `reflect.DeepEquals`, but in this case it would not work. Two instances of `time.Time` can represent the same time instant, but in different time zones, so the values in the struct would be different.

Reply:
ACK